go/cmd/betabot/main.go

```diff
@@ -57,64 +62,326 @@ func betabot() error {
 		qrterminal.GenerateHalfBlock(res.HTMLURL, qrterminal.L, os.Stdout)
 	}
 
+	var wg sync.WaitGroup
+	// request contact in flag
+	{
+		if *contact != "" {
+			splits := strings.Split(*contact, ",")
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				for _, v := range splits {
+					parseRet, err := messengerClient.ParseDeepLink(ctx, &bertymessenger.ParseDeepLink_Request{Link: v})
+					if err != nil {
+						log.Printf("contact was not valid")
+						return
+					}
+					{
+						om, err := proto.Marshal(&bertymessenger.ContactMetadata{DisplayName: *displayName})
+						if err != nil {
+							cancel()
+							return
+						}
+						_, err = messengerClient.SendContactRequest(ctx, &bertymessenger.SendContactRequest_Request{BertyID: parseRet.BertyID, OwnMetadata: om})
+						if err != nil {
+							cancel()
+							return
+						}
+					}
```

Comment:
just a note, nothing to do in this PR, but I think we can add an helper in the bertymessenger package for this